### ISSUE TITLE
Meaning of negative deltas

### ISSUE THREAD

**Issue by Ze-Yang** [clovaai/ECLIPSE#13]
I notice from the codes that the deltas is set to [0.5,-0.4,-0.4] for 50-50 setting. May I know the meaning of negative deltas? From my view, I suppose the delta should be an adjusting factor and therefore should be greater than 0.

**Comment by Ze-Yang** [comment]
Negative delta means calibration the class logits without aggregation of other class knowledge. The authors empirically found that this strategy is effective when the number of incremental steps is small (e.g., 100-50).
